src/nflxmultisubs.js

```diff
@@ -1110,6 +1141,24 @@ class ManifestManagerBase {
   enumManifest() {}
   getManifest(movieId) {}
   saveManifest(manifest) {}
+
+  mergeManifests(manifest, newManifest) {
+    newManifest.timedtexttracks.filter(t => t.hydrated && t.rank!=-1).forEach(track => {
+      for (let i in manifest.timedtexttracks) {
+        if (manifest.timedtexttracks[i].new_track_id == track.new_track_id) {
+          if (!manifest.timedtexttracks[i].hydrated) {
+            // keep original rank to avoid rank conflict
+            track.rank = manifest.timedtexttracks[i].rank;
+            manifest.timedtexttracks[i] = track;
+            console.log(`Manifest for subtitle "${track.language}" updated`);
+          }
+          return;
+        }
+      }
+      console.log(`Cannot find track ${track.new_track_id} to merge with`);
+    })
```

Comment:
Could you add a try block around this? If this somehow breaks in the future, it may prevent the player from breaking completely.

PS: if you do find a workaround for the other issues above with the `secondaryLanguageLastUsed`, also put that code in a try block so that the original functionality may keep working in case it breaks.